src/client/pages/Offer/ProductSelector/NewProductSelector.tsx

```diff
@@ -1,8 +1,57 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import { QuoteCartQuery } from 'data/graphql'
+import { getAllQuotes } from 'api/quoteCartQuerySelectors'
+import { useTextKeys } from 'utils/textKeys'
+import { useLocalizeNumber } from 'l10n/useLocalizeNumber'
+import {
+  ContainerWrapper,
+  Container,
+  HeadingWrapper,
+  HeadingBlack,
+  Column,
+  ColumnSpacing,
+} from '../components'
+import { NewSelector } from './NewSelector'
 
 export type NewProductSelectorProps = { quoteCartQueryData: QuoteCartQuery }
 
-export const NewProductSelector = () => {
-  return <div>New Product Selector</div>
+export const NewProductSelector = ({
+  quoteCartQueryData,
+}: NewProductSelectorProps) => {
+  const textKeys = useTextKeys()
+  const localizedPerMonth = textKeys.SIDEBAR_PRICE_SUFFIX_INTERVAL()
+  const localizeNumber = useLocalizeNumber()
+
+  const products = useMemo(() => {
+    return getAllQuotes(quoteCartQueryData).map(
+      ({ insuranceType, displayName, description, price }) => {
+        return {
+          id: insuranceType,
+          name: displayName,
+          description,
+          price: `${localizeNumber(Math.round(Number(price.amount)))} ${
```

Comment:
Should we do this client side?
When do we ever need to round a price -- they already take care of that in the backend, no?

Reply:
I see. I've based this logic from [`InsuranceSelector`](https://github.com/HedvigInsurance/web-onboarding/blob/master/src/client/pages/Offer/InsuranceSelector/index.tsx#L111). But we don't need it then.